tests/ldap/test_connector.py

```diff
@@ -159,3 +163,40 @@ def test_functional_account_mocked(ldap_mocker: LDAPMocker) -> None:
         "sAMAccountName": "XY",
     }
     assert unit.model_dump(exclude_none=True) == expected
+
+
+def test_fetch_backoff_reconnect(monkeypatch: MonkeyPatch) -> None:
+    # Patch _setup_connection to return different MagicMock connections each time
```

Comment:
this comment is describing line 190 not 170, right?

Reply:
rephrased in e12869fa4ebcd26e48bcb2c7aa596eff88a1969f